src/modules/commander/state_machine_helper.cpp

```diff
@@ -1145,25 +1145,23 @@ void battery_failsafe(orb_advert_t *mavlink_log_pub, const vehicle_status_s &sta
 		// FALLTHROUGH
 		case LOW_BAT_ACTION::RETURN_OR_LAND:
 
-			// let us send the critical message even if already in RTL
-			if (TRANSITION_DENIED != main_state_transition(status, commander_state_s::MAIN_STATE_AUTO_RTL, status_flags,
-					internal_state)) {
-				mavlink_log_critical(mavlink_log_pub, "%s, Executing RTL", battery_critical);
+			if (status_flags.condition_global_position_valid && status_flags.condition_home_position_valid) {
+				internal_state->main_state = commander_state_s::MAIN_STATE_AUTO_RTL;
+				internal_state->timestamp = hrt_absolute_time();
+				mavlink_log_critical(mavlink_log_pub, "%s, executing RTL", battery_critical);
```

Comment:
I saw that there are a few more `mavlink_log_` messages in state_machine_helper, that have this issue with the capitalization. Do you want to change all of them in a separate issue and leave it capitalized here, such that the various messages are at least congruently wrong? :smile:

Reply:
Thanks, yes let me decapitalize (is that a word?) the rest.

Reply:
Thanks :grin: 
_decapitalize_ can also mean to remove capital, but then again google image searches of PX4 show a gallery of handguns. So I don't think we have to worry about this :sweat_smile: